src/AirtableClient.php

```diff
@@ -9,13 +9,13 @@
  */
 class AirtableClient
 {
-    private $key;
-    private $id;
+    private string $key;
+    private string $id;
     
-    public function __construct($key, $id)
+    public function __construct(string $airTableApiKey, string $airTableId)
     {
-        $this->key = $key;
-        $this->id = $id;
+        $this->key = $airTableApiKey;
```

Comment:
Should be rename me too

Reply:
Agreed. Properties have been renamed too